docker-compose.yml

```diff
@@ -161,17 +227,22 @@ services:
       elasticsearch:
         condition: service_healthy
     volumes:
-      - ./config:/etc/georchestra
+      - georchestra_datadir:/etc/georchestra
       - geonetwork_datadir:/mnt/geonetwork_datadir
     environment:
       - DATA_DIR=/var/lib/geonetwork_data
-      - ES_HOST=elasticsearch
+      - ES_FEATUREPROXY_TARGETURI=${ES_HOST}
       - ES_PROTOCOL=http
       - ES_PORT=9200
-      - KB_URL=http://kibana:5601
+      - KB_URL=${KB_HOST}
       - JAVA_OPTIONS=-Duser.home=/tmp/jetty -Dgeorchestra.datadir=/etc/georchestra -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005 -Dorg.eclipse.jetty.annotations.AnnotationParser.LEVEL=OFF
       - XMS=256M
       - XMX=6G
+      - GEORCHESTRA_CONSOLE_HOST=${CONSOLE_HOST}
```

Comment:
Switched from `GEORCHESTRA_CONSOLE_HOST` to `CONSOLE_HOST`:
- https://github.com/georchestra/datadir/commit/feb57fbc9722f6113cbbc7740da811863bb20c99
- https://github.com/georchestra/helm-georchestra/commit/58319b3f61c3e71ffd3e7e5847ef951b15ba84f9